site/en/r2/tutorials/load_data/text.ipynb

```diff
@@ -0,0 +1,706 @@
+      },
+      "cell_type": "markdown",
+      "source": [
+        "This tutorial provides an example of how to use `tf.data.TextLineDataset` to load examples from text files. `TextLineDataset` is designed to create a dataset from a text file, in which each example is a line of text from the original file. This is potentially useful for any text data that is primarily line-based (for example, poetry or error logs).\n",
+        "\n",
+        "In this tutorial, we'll use three different English translations of the same work, Homer's Illiad, and train a model to identify the translator given a single line of text."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "fgZ9gjmPfSnK",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Setup"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "I4dwMQVQMQWD",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!pip install tensorflow==2.0.0-alpha0"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "baYFZMW_bJHh",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "from __future__ import absolute_import, division, print_function \n",
+        "\n",
+        "import requests\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_datasets as tfds\n"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "YWVWjyIkffau",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The texts of the three translations are by:\n",
+        "\n",
+        " - [William Cowper](https://storage.googleapis.com/download.tensorflow.org/data/illiad/cowper.txt)\n",
+        " \n",
+        " - [Edward, Earl of Derby](https://storage.googleapis.com/download.tensorflow.org/data/illiad/derby.txt)\n",
+        " \n",
+        "- [Samuel Butler](https://storage.googleapis.com/download.tensorflow.org/data/illiad/butler.txt)\n",
+        "\n",
+        "The text files used in this tutorial have undergone some typical preprocessing tasks, mostly removing stuff — document header and footer, line numbers, chapter titles. Download these lightly munged files locally. "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "hRI0ZlhTgQ8Y",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "DIRECTORY_URL = 'https://storage.googleapis.com/download.tensorflow.org/data/illiad/'\n",
+        "FILE_NAMES = ['cowper.txt', 'derby.txt', 'butler.txt']\n",
+        "\n",
+        "for name in FILE_NAMES:\n",
+        "  r = requests.get(\"\".join([DIRECTORY_URL, name]), allow_redirects=True)\n",
+        "  with open(name, 'w') as f:\n",
+        "    f.write(r.content)"
```

Comment:
Getting an error here when run in Colab w/ py3: `TypeError: write() argument must be str, not bytes`
Maybe open with `'wb'`?